lib/ansible/modules/system/iocage.py

```diff
@@ -0,0 +1,615 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2019, Lars Liedtke <liedtke@punkt.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
```

Comment:
Should start with v1.0.

```suggestion
    'metadata_version': '1.0',
```

Reply:
I was not sure about this value, wanted to look it up and forgot